CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
 # Changelog
 
 ## [0.12.0](https://github.com/brave/browser-laptop/releases/v0.12.0dev)
-- (todo)
+ - Added shortcut to view page source ([#3685](https://github.com/brave/browser-laptop/issues/3685))
+ - Added extra separator under bookmark menu ([#3666](https://github.com/brave/browser-laptop/issues/3666))
+ - Added an option to temporarily disable mixed content protection enhancement security ([#3443](https://github.com/brave/browser-laptop/issues/3443))
```

Comment:
This one is more important than bookmark double separator, and actually it was a bugfix to remove an extra separator so that should be under fixed pls and not very important so towards the bottom.